src/content.config.mjs

```diff
@@ -20,6 +20,7 @@ const specialEvents = defineCollection({
         url: 'https://docs.google.com/spreadsheets/d/e/2PACX-1vQ9lwK4u_pUe90Ff5o-2OpGSAjMv2gN-CQF1vTBbwfZgKFLYMuHhW8rjp4fsyhhL6R9ANYalYBQRjmh/pub?gid=548759105&single=true&output=csv',
     }),
     schema: () => z.object({
+        'Id': z.optional(z.string()),
```

Comment:
Can we make the Sheet like...auto-populate the ID? Generate a UUID or something?